src/components/Page_404/Error.css

```diff
@@ -0,0 +1,90 @@
+*{
+margin:0;
+padding:0;
+background-color:white;
+width: 100% vw;
+height: 100% vh;
+}
+.style{
+margin-left: 25%;
+margin-right: 33%;
+font-size:24px;
+font-weight:bold;
+color:darkgrey;
+padding-top: 50px; 
+padding-bottom:0px;
+text-align:center;
+}
+.text{
+font-size: 20px;
+text-align: center;
+}
+img {
+width: 300px;
+height: 200px;
+}
+#styling{
+height: 70px;
+width: 90px;
+float: left;
+}
+@keyframes colorchange{
+20%{ color:blue;}
+40%{ color:green;}
+60%{ color:red;}
+80%{ color:black;}
+}
+.button{
+margin:7px;
+padding:15px 25px;
+font-size:24px;
+text-align:center;
+cursor:pointer;
+font-weight:bold;
+outline:none;
+color:indigo;
+background-color:lightgrey;
+border:none;
+border-radius:5px;
+animation:colorchange 3s infinite alternate;
```

Comment:
Change this to make it look more appropriate and perform animation someplace else instead of text. It doesn't look good!